panel/dist/css/dataframe.css

```diff
@@ -36,6 +36,11 @@ table.panel-df {
 }
 
 .panel-df tr:hover {
-    background: lightblue !important;
+    background: #e1f5fe !important;
```

Comment:
The color is prettier (my preference) and aligned with Jupyter

Reply:
Agreed.

Reply:
I was thinking about wheyher we should set max_rows and max_cols to some number to minimize the risk of crashing the notebook?